lib/geodb.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var geodb = require('fxa-geodb')()
+
+module.exports = function (log) {
+  // thin wrapper around geodb, to help log the accuracy
+  return function (ip, request) {
+    return geodb(ip)
+            .then(function (loc) {
+              var logEventPrefix = 'fxa.location.accuracy.'
+              var logEvent = ''
+              var accuracy = loc.accuracy
+              if (accuracy > 200) {
```

Comment:
make `200` and other numbers "constants" and put them above `module.exports`. Also what unit are they ? make unit as part of variable name. like `ACCURACY_MAX_KM` and `MIN`